packages/fxa-payments-server/backstage.yaml

```diff
@@ -12,10 +12,10 @@ metadata:
     - node
     - react
   links:
-    - url: https://subscriptions.firefox.com/
+    - url: https://payments.firefox.com/
       title: Production Subscription Platform
       type: website
-    - url: https://payments-server.allizom.org/
+    - url: https://payments-next.allizom.org/
```

Comment:
this change is unrelated to the PR, but the links were outdated. Not sure if this file has any actual impact though.